presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/BinaryColumnStatistics.java

```diff
@@ -31,9 +31,11 @@ public class BinaryColumnStatistics
     public BinaryColumnStatistics(
```

Comment:
All BlaBlaColumnStatistics constructors need to be adjusted for the new size fields.